node/nodeRunner.go

```diff
@@ -1242,7 +1238,7 @@ func (nr *nodeRunner) CreateManagedProcessComponents(
 		ChainRunType:           common.ChainRunTypeRegular,
 	}
 
-	return nr.createManagedProcessComponentsMethod(processArgs)
+	return nr.createManagedProcessComponents(processArgs)
 }
 
 func (nr *nodeRunner) createManagedProcessComponents(processArgs processComp.ProcessComponentsFactoryArgs) (mainFactory.ProcessComponentsHandler, error) {
```

Comment:
I think also this could be removed all the method content should be move in L1241 if we want to revert to the original code

Reply:
reverted